recipe/build.sh

```diff
@@ -1,11 +1,11 @@
-mkdir -p build_conda
-cd build_conda
-rm -f CMakeCache.txt
-
 export CMAKE_PATH_PREFIX=${PREFIX}
 
 cmake ${CMAKE_ARGS} ../ \
       -DENABLE_EMBREE=on \
-      -DENABLE_OPTIX=off
+      -DENABLE_OPTIX=off \
+      -B build \
+      -S . \
+      -G Ninja
 
-make install
+cmake --build build
+cmake --install build
```

Comment:
Do we want to specify Release configurations here, for consistency?
```suggestion
cmake --build build --config Release
cmake --install build --config Release
```

Reply:
Sure, we can. They should be no-ops here - but I was surprised that they were required in the windows build. The Ninja generator must work differently on Windows, because it ignored the CMAKE_BUILD_TYPE and built a Debug build until I explicitly added `--config Release`.